client/blocks/sideload-a8c-analytics-banners/index.tsx

```diff
@@ -0,0 +1,105 @@
+import { loadScript } from '@automattic/load-script';
+import { translate } from 'i18n-calypso';
+import { useEffect } from 'react';
+import { debug, A8C_ANALYTICS_CACHE_VERSION } from 'calypso/lib/analytics/ad-tracking/constants';
+import { loadTrackingScripts } from 'calypso/lib/analytics/ad-tracking/load-tracking-scripts';
+
+type A8CAnalyticsConfig = {
+	cookieBanner: unknown; // WIP: Import typing from a8c-analytics.js
+};
+
+declare global {
+	interface Window {
+		a8cAnalyticsConfig: A8CAnalyticsConfig;
+	}
+}
+
+const a8cAnalyticsConfig: A8CAnalyticsConfig = {
+	cookieBanner: {
+		version: '2',
+		cssZIndex: 50001,
+		skipBanner: false,
+		cookieDomain: null,
+		v1CookieName: 'sensitive_pixel_option',
+		v2CookieName: 'sensitive_pixel_options',
+		v1Text: translate( 'Our websites and dashboards use cookies.' ),
+		v2Text: translate(
+			'We and our partners process your personal data (such as browsing data, IP addresses, cookie information, and other unique identifiers) based on your consent and/or our legitimate interest to optimize our website, marketing activities, and your user experience.'
+		),
+		v2OptionsText: translate(
+			// translators: %(privacyUrl|cookieUrl)s is a URL like <https://automattic.com/privacy/> or <https://automattic.com/cookies/>
+			'Your privacy is critically important to us. We and our partners use, store, and process your personal data to optimize: our <strong>website</strong> such as by improving security or conducting analytics, <strong>marketing activities</strong> to help deliver relevant marketing or content, and your <strong>user experience</strong> such as by remembering your account name, language settings, or cart information, where applicable. You can customize your cookie settings below. Learn more in our <a href="%(privacyUrl)s" target="_blank">Privacy Policy</a> and <a href="%(cookieUrl)s" target="_blank">Cookie Policy</a>.',
```

Comment:
There's a special syntax for HTML or components in localized strings, using the `components` argument. Here's an example:

https://github.com/Automattic/wp-calypso/blob/511d41a06550a50ea2e7dd8c72367e6d85b2cd00/apps/notifications/src/panel/templates/body.jsx#L139